lib/auth/apiserver.go

```diff
@@ -97,6 +97,9 @@ func NewAPIServer(config *APIConfig) http.Handler {
 	srv.POST("/:version/users/:user/ssh/authenticate", srv.withAuth(srv.authenticateSSHUser))
 	srv.GET("/:version/users/:user/web/sessions/:sid", srv.withAuth(srv.getWebSession))
 	srv.DELETE("/:version/users/:user/web/sessions/:sid", srv.withAuth(srv.deleteWebSession))
+
+	// User signup tokens.
+	srv.GET("/:version/signuptokens", srv.withAuth(srv.getSignupTokens))
```

Comment:
in Gravity we do not use this method directly. Existing signupTokens are used implicitly when Web UI calls for 'GET/getUsers' to get a list of local users. If signup token exists, the user status will be 'invited' with other properties populated from corresponding signup token.